example-app.conf

```diff
@@ -20,6 +20,9 @@ ELASTICSEARCH_ENDPOINT = 'https://search-api.test.hubmapconsortium.org'
 ENTITY_API_BASE = 'https://entity-api.test.hubmapconsortium.org'
 ASSETS_ENDPOINT = 'https://assets.test.hubmapconsortium.org'
 
+# Assay type service endpoint
+TYPE_SERVICE_ENDPOINT = 'https://search-api.test.hubmapconsortium.org'
```

Comment:
It's a separate endpoint in the other repos in which the service is referenced, and eventually it will have to be the separate ontology endpoint.  I thought about just using ELASTICSEARCH_ENDPOINT but decided it was preferable to add the new config parameter sooner rather than later.  Also, during this interim stage where the version of the ELASTICSEARCH_ENDPOINT which includes the type service API has not yet been pushed to TEST, having the separate config parameter allows the type client to be pointed at the working version on DEV.